shell/edit/monitoring.coreos.com.prometheusrule/GroupRules.vue

```diff
@@ -28,6 +28,26 @@ export default {
     },
   },
 
+  data(props) {
+    const defaultAlert = {
+      alert:  '',
+      expr:   '',
+      for:    '0s',
+      labels: {
+        severity:  'none',
+        namespace: 'default'
+      },
+    };
+
+    if (!this.value.rules) {
+      this.value['rules'] = [defaultAlert];
+    } else if ( this.value.rules.length === 0) {
+      this.value['rules'] = [...this.value.rules, defaultAlert];
```

Comment:
Pr looks good, I'm just being nitpicky: is there a reason to spread `this.value.rules` when the array is empty?

Reply:
No, this is a mistake. I meant to only spread it if there was already a label in there. will fix